packages/dataviews/src/add-filter.js

```diff
@@ -107,50 +110,45 @@ export default function AddFilter( { filters, view, onChangeView } ) {
 							>
 								<WithSeparators>
 									<DropdownMenuGroup>
-										{ filter.elements.map( ( element ) => (
-											<DropdownMenuItem
-												key={ element.value }
-												role="menuitemradio"
-												aria-checked={
-													activeElement?.value ===
-													element.value
-												}
-												prefix={
-													activeElement?.value ===
-														element.value && (
-														<Icon icon={ check } />
-													)
-												}
-												onSelect={ ( event ) => {
-													event.preventDefault();
-													onChangeView(
-														( currentView ) => ( {
-															...currentView,
+										{ filter.elements.map( ( element ) => {
```

Comment:
Three refactorings (no behavioral changes) in this component:

- extracted `otherFilters`
- extracted `isActive`
- used `onChangeView({ ... })` instead of `onChangeView( ( currentView ) => { ... })`.